src/applications/vaos/tests/e2e/page-objects/TypeOfCarePageObject.js

```diff
@@ -20,17 +20,17 @@ export class TypeOfCarePageObject extends PageObject {
     return super.assertUrl(
       {
         url: '/type-of-care',
-        breadcrumb: 'Schedule an appointment',
+        breadcrumb: 'What type of care do you need?',
       },
       { timeout: 10000 },
     );
   }
 
   selectTypeOfCare(label) {
-    cy.findByLabelText(label)
-      .as('radio')
-      .focus();
-    cy.get('@radio').check();
+    cy.get('va-radio')
+      .shadow()
+      .get(`va-radio-option[value=${label}]`)
+      .click();
```

Comment:
Why the change? Ideally, it's better to write test by what the user sees on the screen as opposed to implementation details that might change.

Reply:
`findByLabelText` doesn't work with the `VaRadioField` widget, I didn't investigate much but it might have something to do with the shadow DOM. I took a look at how others wrote tests when using the `VaRadioField` and this is how they interact with it. Selecting by value also works consistently across E2E test and unit tests.